build_tools/travis/install.sh

```diff
@@ -53,11 +53,13 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl flake8
+            cython=$CYTHON_VERSION libgfortran mkl flake8 \
+            pandas=$PANDAS_VERSION
```

Comment:
I came up with two ways of making PANDAS_VERSION conditional. Which one of these is the right way? If not then how can I proceed.
```
if [[ "$INSTALL_MKL" == "true"]]; then
    conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
            numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
            cython=$CYTHON_VERSION libgfortran mkl flake8 \
            if [[-v $PANDAS_VERSION]]; then
                pandas=$PANDAS_VERSION
            fi
```

```
if [[ "$INSTALL_MKL" == "true" &&  [-v $PANDAS_VERSION] ]; then
    conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
            numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
            cython=$CYTHON_VERSION libgfortran mkl flake8 \
            pandas=$PANDAS_VERSION
        
elif [[ "$INSTALL_MKL" == "true" ]]
        conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
            numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
            cython=$CYTHON_VERSION libgfortran mkl flake8 \
fi
```